src/m365/teams/commands/app/app-list.ts

```diff
@@ -44,28 +43,32 @@ class TeamsAppListCommand extends GraphItemsListCommand<TeamsApp> {
       return Promise.resolve(args.options.teamId);
     }
 
-    const teamRequestOptions: any = {
-      url: `${this.resource}/v1.0/me/joinedTeams?$filter=displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/groups?$filter=displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
       headers: {
         accept: 'application/json;odata.metadata=none'
       },
       responseType: 'json'
     };
 
     return request
-      .get<{ value: Team[] }>(teamRequestOptions)
+      .get<{ value: [{ id: string, resourceProvisioningOptions: string[] }] }>(requestOptions)
```

Comment:
Instead of defining our own type, we should load the type from the MS Graph types package. It seems though that the `Group` entity is missing the `resourceProvisioningOptions` property, so for now we could create a custom property that extends `Group` and when the typings are fixed, we'd replace it.